forte/processors/nltk_processors.py

```diff
@@ -97,6 +102,38 @@ def penn2morphy(penntag: str) -> str:
         return 'n'
 
 
+class NLTKChunker(PackProcessor):
+    r"""A wrapper of NLTK chunker.
+    """
+    def __init__(self):
+        super().__init__()
+        self.chunker = None
+        self.token_component = None
+
+    # pylint: disable=unused-argument
+    def initialize(self, resource: Resources, configs: HParams):
+        self.chunker = RegexpParser(configs.pattern)
+
+    @staticmethod
+    def default_hparams():
+        r"""This defines a basic Hparams structure for NLTKChunker.
+        """
+        return {
+            'pattern': 'NP: {<DT>?<JJ>*<NN>}',
+        }
+
+    def _process(self, input_pack: DataPack):
+        for sentence in input_pack.get(Sentence):
+            token_entries = list(input_pack.get(entry_type=Token,
+                                                range_annotation=sentence,
+                                                component=self.token_component))
+            tokens = [(token.text, token.pos) for token in token_entries]
+            cs = self.chunker.parse(tokens)
+            iob_tagged = tree2conlltags(cs)
+            for token, iob_tag in zip(token_entries, iob_tagged):
+                token.set_fields(chunk=iob_tag[2])
```

Comment:
Let's create a Phrase object in our base_ontology to hold the chunks, that way the end user don't need to go over the IOB tags.